modules/core/src/main/scala/org/scalasteward/core/sbt/SbtAlg.scala

```diff
@@ -130,7 +136,11 @@ object SbtAlg {
           updates <- withTemporarySbtDependency(repo) {
             exec(sbtCmd(commands), repoDir).map(parser.parseSingleUpdates)
           }
-        } yield updates
+          originalDependencies <- getOriginalDependencies(repo)
```

Comment:
This would be the second sbt call in `getUpdatesForRepo`. Since sbt takes a long time to load, this will nearly double the time needed to execute this method. And this is already one of the most expensive calls when working on a repository. I guess my public instance of Scala Steward would take over one hour more for a full run just for this additional sbt call.

I see two options to speed this up:
1. Use `RepoCacheAlg` here to get the dependencies of a repo.
1. Only call sbt once to get updates and dependencies.

I hadn't the time to thought about the full implications of either of these approaches.

Reply:
I picked first option in https://github.com/fthomas/scala-steward/pull/676/commits/b51a779a506cffdd4c9350100231db40c92a4128 and it seems to work well.
Before test run, I cleared out `workspace`.

Reply:
Thanks! In the future we might want to have a new method that checks if the cache is up-to-date and refreshes it if is outdated before returning dependencies. Because right now, the caches are only up-to-date if Scala Steward is run with `--prune-repos=true`. But this OK for now.